app/src/test/java/org/mozilla/fenix/wallpapers/LegacyWallpaperFileManagerTest.kt

```diff
@@ -0,0 +1,111 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.wallpapers
+
+import kotlinx.coroutines.test.UnconfinedTestDispatcher
+import kotlinx.coroutines.test.runTest
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertTrue
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.TemporaryFolder
+import java.io.File
+
+class LegacyWallpaperFileManagerTest {
```

Comment:
anything to call-out for differences to this batch of tests, or is this just the `legacy` migration?

Reply:
The legacy files should be straight renames. The `WallpaperDownloader` tests are new, and the new `WallpaperFileManagerTest` has the updated paths but I think that's it.